control/statesp.py

```diff
@@ -1746,87 +1630,6 @@ def ss(*args, **kwargs):
     return sys
 
 
-# Convert a state space system into an input/output system (wrapper)
```

Comment:
I can put `ss2io` and `tf2io` back in with a deprecation warning, but then we have to carry them around until v0.11 (since we don't change the user interface at the patch level).  I'd be interested in thoughts on this from others.

Reply:
I have a few of these sprinkled around in examples and demos that I wrote back when `ss` and `tf` systems didn't have signal names (and when `tf` didn't correctly keep signal names if the system was static and slycot was not installed : ) 

Just a bug that students could encounter, and it doesnt seem to be a big burden to keep them around. It might be enough to make them simple wrappers of `ss` and `tf`, respectively, in the corresponding modules, that works correctly but raises a deprecation warning. We are usually pretty good at avoiding breaking old code .. at least for awhile.

Reply:
Let me give it a quick shot and then perhaps you can run on your existing code to see if everything works.  My sense is that we are going to break things with this release (part of the point) and so we may as well make changes all at once.  But I agree that we should also deprecate things properly, rather than just dropping code that hasn't been marked as such.

The new code will indeed be just a quick wrapper (since the object that is being passed to ss2tio or tf2io is already and I/O system...).